packages/multimedia/ffmpeg/package.mk

```diff
@@ -3,14 +3,13 @@
 # Copyright (C) 2017-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="ffmpeg"
-PKG_LICENSE="LGPLv2.1+"
-PKG_SITE="https://ffmpeg.org"
-PKG_DEPENDS_TARGET="toolchain zlib bzip2 openssl speex"
-PKG_LONGDESC="FFmpeg is a complete, cross-platform solution to record, convert and stream audio and video."
-
 PKG_VERSION="4.4.1"
 PKG_SHA256="eadbad9e9ab30b25f5520fbfde99fae4a92a1ae3c0257a8d68569a4651e30e02"
+PKG_LICENSE="LGPL-3.0-or-later"
```

Comment:
that should be GPL-3.0, not LGPL-3.0 as we're configuring ffmpeg with `--enable-gpl --enable-version3`
https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/multimedia/ffmpeg/package.mk#L157-L158

From the ffmpeg docs it's not quite clear if that means it's 3.0-only or 3.0-or-later - LICENSE.md only mentions v3 (version 2 mentions are explicity described as v2+)

And we should probably fix PKG_LICENSE of kodi, too, currently it's just `GPL` where it should be GPL-3.0 as well as it's being linked with GPL3 ffmpeg